src/Tools/BuildActionTelemetryTable/BuildActionTelemetryTable.csproj

```diff
@@ -21,6 +21,7 @@
 
   <!-- These are dependencies that are never deployed but that we need to reflect over the Microsoft.CodeAnalysis.EditorFeatures assemblies -->
   <ItemGroup>
+    <PackageReference Include="Microsoft.IO.Redist" Version="$(MicrosoftIORedistVersion)" NoWarn="NU1701" />
```

Comment:
I had to add this to suppress NU1701, which evidently you've already done plenty of here.
Microsoft.IO.Redist is a newly expressed dependency of Microsoft.VisualStudio.Imaging.